lib/WorkflowViralrecon.groovy

```diff
@@ -1,59 +0,0 @@
-//
```

Comment:
I guess this file was renamed? But did not see the new script on this PR :open_mouth:

Reply:
This came in via the template sync but we have two separate workflows to process `illumina` and `nanopore` data so we don't need one named by the pipeline itself which is why I just deleted it.

Reply:
OK, I see! thanks!